packages/gamut-labs/src/TabsExperimental/Tab.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, TextButton } from '@codecademy/gamut';
+import styled from '@emotion/styled';
+import React from 'react';
+
+import { TabProps } from './types';
+
+type TabButtonProps = Pick<TabProps, 'isActiveTab'>;
+
+const TabButton = styled(TextButton)<TabButtonProps>`
+  border: 0;
+  border-bottom: ${({ theme, isActiveTab }) => {
+    let borderColor = 'transparent';
+    if (isActiveTab) {
+      if (theme.mode === 'light') {
+        borderColor = theme.colors.blue;
+      } else {
+        borderColor = theme.colors.primary;
+      }
+    }
+    return `0.1875rem ${borderColor} solid`;
+  }};
+  border-bottom-left-radius: 0px;
+  border-bottom-right-radius: 0px;
+  font-weight: ${({ theme, isActiveTab }) =>
+    isActiveTab ? theme.fontWeight[700] : theme.fontWeight[400]};
+  font-size: ${({ theme }) => theme.fontSize[18]};
+  text-overflow: ellipsis;
+  color: ${({ theme, isActiveTab }) => {
+    let textColor: string = theme.colors.text;
```

Comment:
Nit: TypeScript is smort

```suggestion
    let textColor = theme.colors.text;
```

Reply:
I actually get a ts warning if I don't do this: `Type '"var(--color-blue)"' is not assignable to type '"var(--colors-text)"'.`

Reply:
This doesn't matter anymore because I got `states` to work

Reply:
Ooh yay!